script/run-container.sh

```diff
@@ -125,6 +125,11 @@ container_tnf_kubeconfig_volumes_cmd_args=$(printf -- "-v %s " "${container_tnf_
 CONTAINER_TNF_DOCKERCFG=$(join_paths "${container_tnf_dockercfg_paths[@]}")
 container_tnf_dockercfg_volumes_cmd_args=$(printf -- "-v %s " "${container_tnf_dockercfg_volume_bindings[@]}")
 
+# Safeguard against an empty variable
+if [[ -n $CONTAINER_TNF_DOCKERCFG ]]; then
```

Comment:
Shouldn't you use "-z" instead? From[ bash manual](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html):
```
-z string
True if the length of string is zero.

-n string
string
True if the length of string is non-zero.
```

Reply:
I'd use single bracket:
```
# Sets default if it is empty.
[ -z "$CONTAINER_TNF_DOCKERCFG" ] && CONTAINER_TNF_DOCKERCFG="$HOME/.docker/config"
```

Reply:
Okay, I changed it to: 

```
if [ -z $CONTAINER_TNF_DOCKERCFG ]; then
	CONTAINER_TNF_DOCKERCFG="NA"
fi
```